### PR TITLE
Release v5.4.1

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Review by nuttycom — APPROVED** [review]
utACK

File: qa/zcash/postponed-updates.txt [hunk 1]
```diff
@@ -46,6 +50,7 @@ native_rust 1.65.0 2023-03-01
 native_rust 1.66.0 2023-03-01
 native_rust 1.66.1 2023-03-01
 native_rust 1.67.0 2023-03-01
+native_rust 1.67.1 2023-03-01
```
Comment (daira) [inline qa/zcash/postponed-updates.txt:53]: Checked that this isn't important to upgrade to.

File: qa/zcash/postponed-updates.txt [hunk 2]
```diff
@@ -4,19 +4,23 @@
 # bdb 18.1.40 2020-09-01
 #
 
+native_zstd 1.5.4 2023-03-01
```
Comment (daira) [inline qa/zcash/postponed-updates.txt:7]: This appears to be [just performance improvements](https://www.phoronix.com/news/Zstandard-1.5.4-Released), which are not important.

File: qa/zcash/postponed-updates.txt [hunk 3]
```diff
@@ -4,19 +4,23 @@
...
 native_cxxbridge 1.0.88 2024-02-01
 native_cxxbridge 1.0.89 2024-02-01
+native_cxxbridge 1.0.90 2024-02-01
 rustcxx 1.0.84 2024-02-01
 rustcxx 1.0.85 2024-02-01
 rustcxx 1.0.86 2024-02-01
 rustcxx 1.0.87 2024-02-01
 rustcxx 1.0.88 2024-02-01
 rustcxx 1.0.89 2024-02-01
+rustcxx 1.0.90 2024-02-01
```
Comment (daira) [inline qa/zcash/postponed-updates.txt:23]: > Prevent new clippy lint extra_unused_type_parameters from triggering inside generated code
So not important to upgrade to.

**Review by daira — APPROVED** [review]
utACK

**Pull request merged**